src/constants.js

```diff
@@ -32,6 +32,7 @@ export const ONBOARDING_ERRORS = [
 export const IS_STARTED_STATUS = (status) => [ExamStatus.STARTED, ExamStatus.READY_TO_SUBMIT].includes(status);
 export const IS_INCOMPLETE_STATUS = (status) => INCOMPLETE_STATUSES.includes(status);
 export const IS_ONBOARDING_ERROR = (status) => ONBOARDING_ERRORS.includes(status);
+export const IS_PROCTORED_STATUS = (status) => IS_STARTED_STATUS(status) || status === ExamStatus.READY_TO_START;
```

Comment:
could you add a comment explaining the use case for this group of statuses? The statuses are also applicable to timed exams, so I was ~slightly confused reading the name at first (before looking at how it's used in the code)